assets/js/components/notifications/BannerNotifications.js

```diff
@@ -70,7 +73,7 @@ export default function BannerNotifications() {
 					{ isAuthenticated && <CoreSiteBannerNotifications /> }
 					{ dashboardSharingEnabled && <ModuleRecoveryAlert /> }
 					{ ga4ActivationBannerEnabled && <ActivationBanner /> }
-					{ twgEnabled && (
+					{ twgEnabled && twGModuleAvailable && (
```

Comment:
This notification should only be shown when TwG is connected. As it is, it's triggering a request for the `supporter-wall-prompt` on the dashboard any time the feature is enabled, regardless of the state of the module which is not what we want.

Since a module cannot be connected without also being active and available, this is essentially the only condition we need here.

Reply:
Updated.